### DISCUSSION TITLE
How to reset or add another user in Nextcloud

### DISCUSSION THREAD

**Discussion by diabolo511** [tteck/Proxmox#3445] [Help Needed]
Hey guys!

So I'm having an issue with Nextcloud e.g. logging in on my self-hosted LXC. I was looking up the credentials, but NC is always saying that the Username or Password is wrong. Is there any chance to reset the password so I can log in again?

Cheers

**Comment by Mellowlynx** [comment]
What version are you running?
[NextCloudPi](https://github.com/nextcloud/nextcloudpi#features) for example, has a separate user (NCP)
The NCP admin UI is accessible through https://(IP):4443. You can check if you have that login.

You could also try: https://docs.nextcloud.com/server/latest/admin_manual/configuration_user/reset_admin_password.html
I am not sure if it can work, depends on the setup.

**Comment by diabolo511** [comment]
Hey there!

I'm running the Alpine Container from the scripts! I can try and tell you later!
